bed/bedpe/bedpe.go

```diff
@@ -20,6 +20,13 @@ type BedPe struct {
 	B bed.Bed
 }
 
+type BedPeHalf struct {
+	Chrom      string
+	ChromStart int
+	ChromEnd   int
+	Home       *BedPe
```

Comment:
Can you help me understand why Home is a *BedPe as opposed to a BedPe? Is it so that the pointer can point to the correct value regardless of variable name?

Reply:
The pointer is so that each half of the bedPe can be assigned back to their parent bedPe after overlaps